src/features/events/components/groupedEventsCard.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import moment from 'moment'
+import { FlexColumn } from '../../../components/grid'
+import { media } from '../../../theme/media'
+import constants from '../../../constants'
+import EventListingCard from './eventListingCard'
+
+const DateGroupHeading = styled.h2`
+  margin: 1rem 0;
+
+  ${media.tablet`
+    display: none;
+  `};
+`
+
+const GroupedEventsCards = ({ event, index, events, toLoad }) => {
+  let header
+  const longDayOfMonth = 'dddd D MMM'
+
+  if (index === 0) {
+    header = moment(event.node.startTime).format(longDayOfMonth)
+  } else {
+    const startDate = moment(event.node.startTime).format(constants.dateFormat)
+    const prevStartDate = moment(events[index - 1].node.startTime).format(
+      constants.dateFormat
+    )
+
+    if (startDate !== prevStartDate) {
+      header = moment(event.node.startTime).format(longDayOfMonth)
+    }
+  }
+  return (
+    <FlexColumn
+      width={[
+        1, // 100% between 0px screen width and first breakpoint (375px)
+        1, // 100% between first breakpoint(375px) and second breakpoint (768px)
+        1 / 2, // 50% between second breakpoint(768px) and third breakpoint (1024px)
+        1 / 3, // 33% between third breakpoint(1280px) and fourth breakpoint (1440px)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 85. Maximum allowed is 80.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4658697)